<?xml version="1.0" encoding="UTF-8"?>
<launch>
  	<!-- Add markerDetector node -->
  <node name="markerDetector" pkg="experimental_1" type="markerDetector" output="screen">
   
  </node>

  <!-- Add navlogNode.py node -->
  <node name="navlogNode" pkg="experimental_1" type="nav2.py" output="screen">
    
  </node>

</launch>


